client/webserver/http.go

```diff
@@ -96,18 +95,14 @@ func (s *WebServer) handleRegister(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// TODO: rework register page for paying with other assets.
-	feeAssetID, _ := dex.BipSymbolID("dcr")
-	feeWalletStatus := s.core.WalletState(feeAssetID)
-	feeWalletExists := feeWalletStatus != nil
-	feeWalletOpen := feeWalletExists && feeWalletStatus.Open
+	// feeAssetID, _ := dex.BipSymbolID("dcr")
+	// feeWalletStatus := s.core.WalletState(feeAssetID)
+	// feeWalletExists := feeWalletStatus != nil
+	// feeWalletOpen := feeWalletExists && feeWalletStatus.Open
 
 	switch {
 	case !cArgs.UserInfo.Initialized:
 		data.InitStep = true
-	case !feeWalletExists:
-		data.WalletStep = true
-	case !feeWalletOpen:
-		data.OpenStep = true
```

Comment:
Let's clean up the `registerTmplData` struct, and audit all of frontend for places where these are/were used, and any other obsolete strings.